en/ros/external_position_estimation.md

```diff
@@ -258,7 +274,3 @@ Increase the value of the left stick and the robot will take off,
 put it back to the middle right after. Check if it is able to keep its position.
 
 If it works, you may want to set up an [offboard](offboard_control.md) experiment by sending position-setpoint from a remote ground station.
-
-### VIO First Flight
```

Comment:
This needs to eventually get in. Do not remove it so it stays here as a reference of something that needs to be done.

Reply:
Added it back